test/config/features/FeatureSpec.scala

```diff
@@ -54,4 +54,16 @@ class FeatureSpec extends UnitSpec with GuiceOneAppPerSuite with BeforeAndAfterE
       features.emailVerificationEnabled() shouldBe false
     }
   }
+
+  "The Change Website Feature" should {
```

Comment:
You need to add `features.changeWebsiteEnabled(true)` to the `beforeEach` at the top of the spec which fill fix the inconsistent results issue.